backend/variants/file_export.py

```diff
@@ -421,7 +459,7 @@ def _fetch_gene_scores(self, entrez_ids):
         if self._is_prioritization_enabled():
             try:
                 prio_algorithm = self.query_args.get("prio_algorithm")
-                hpo_terms = tuple(sorted(self.query_args.get("prio_hpo_terms_curated", [])))
+                hpo_terms = tuple(sorted(self.query_args.get("prio_hpo_terms", [])))
```

Comment:
@ahujameg This should not be affected by your changes. It might be that this change re-introduces a bug. Can you please double check if this is change is intentional?